index.js

```diff
@@ -1,89 +1,15 @@
-// Handle page navigation
-
-function handlePageChange(e) {
-  const navLinks = document.getElementsByClassName('nav-link');
-  for (let i = 0; i < navLinks.length; i += 1) {
-    navLinks[i].style.color = '';
-  }
-
-  if (e.target.classList.contains('list')) {
-    document.querySelector('#book-display').style.display = 'block';
-    document.querySelector('#add-book').style.display = 'none';
-    document.querySelector('#contact').style.display = 'none';
-  } else if (e.target.classList.contains('addBook')) {
-    document.querySelector('#book-display').style.display = 'none';
-    document.querySelector('#add-book').style.display = 'block';
-    document.querySelector('#contact').style.display = 'none';
-  } else if (e.target.classList.contains('contactInfo')) {
-    document.querySelector('#book-display').style.display = 'none';
-    document.querySelector('#add-book').style.display = 'none';
-    document.querySelector('#contact').style.display = 'block';
-  }
-}
+import UpdateDate from './modules/luxon.js';
+import handlePageChange from './modules/navigation.js';
 
 document.addEventListener('click', (e) => {
   handlePageChange(e);
 });
 
-// Date
-const timeElement = document.querySelector('.time');
-const dateElement = document.querySelector('.date');
-
-/**
- * @param {Date} date
- */
-function formatTime(date) {
-  const hours12 = date.getHours() % 12 || 12;
-  const minutes = date.getMinutes();
-  const seconds = date.getSeconds();
-  const isAm = date.getHours() < 12;
+// setInterval(() => {
+//   UpdateTime();
+// }, 1000);
```

Comment:
- [ ] [OPTIONAL] You did great work @Fombi-Favour 😃, however, for best practice and clear code, I'll subject you to remove these lines of code 👍